dd-sdk-android-ndk/src/test/cpp/test_generate_backtrace.cpp

```diff
@@ -13,7 +13,7 @@ TEST test_generate_backtrace(void) {
             backtrace.c_str());
     // we don't know if the stack is big enough to cover the required max size of 30
     unsigned int lines_count = backtrace_lines.size();
-    const char *regex = "(\\d+):0[xX][0-9a-fA-F]+(\\s*)(.*)";
+    const char *regex = "(\\d+)(.*)0[xX][0-9a-fA-F](.*)";
```

Comment:
I would keep the `+` after `[0-9a-fA-F]`, it doesn't change anything but that is more clear about what is captured

Reply:
Again...I missed this when I changed the RegEx. Tks for spotting this.